CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ if ( PLUGIN )
     HPCC_ADD_SUBDIRECTORY (plugins/spark "SPARK")
 elseif ( NOT MAKE_DOCS_ONLY )
     HPCC_ADD_SUBDIRECTORY (system)
+    HPCC_ADD_SUBDIRECTORY (cmake_modules)
```

Comment:
I'm not sure how the cmake changes are related to the log level output

Reply:
Ok. will put the change into separate ticket.